www/cadaver/Portfile

```diff
@@ -3,37 +3,29 @@
 PortSystem          1.0
 
 name                cadaver
-version             0.23.3
-revision            3
+version             0.24
+revision            0
 categories          www
 maintainers         {fossekall.de:michael.klein @mklein-de} openmaintainer
 description         Commandline client for DAV
 long_description    cadaver is a command-line WebDAV client for Unix.  It \
                     supports file upload, download, on-screen display, \
                     namespace operations (move/copy), collection creation \
                     and deletion, and locking operations.
-homepage            http://www.webdav.org/cadaver/
+homepage            https://notroj.github.io/cadaver
 platforms           darwin
 license             GPL-2
 
 master_sites        ${homepage}
 
-checksums           rmd160  104f687cfd121b091ba2f509b37574509b4ffabd \
-                    sha256  fd4ce68a3230ba459a92bcb747fc6afa91e46d803c1d5ffe964b661793c13fca \
-                    size 831884
+checksums           rmd160 8efe1502d0be32baaa88eb7a9c15cfaff88e27bd \
+                    sha256 46cff2f3ebd32cd32836812ca47bcc75353fc2be757f093da88c0dd8f10fd5f6 \
+                    size 808069
 
 depends_build       port:pkgconfig
-depends_lib         port:expat \
-                    path:lib/libssl.dylib:openssl \
+depends_lib         path:lib/libssl.dylib:openssl \
                     port:neon \
                     port:readline
 
-configure.args      --with-expat \
```

Comment:
why removing expat?

Reply:
neon already depends on an XML parser (`expat`). Unless I'm misreading cadaver's autoconf code `--with-expat` is a dead option if an external neon is used.

The `./configure` summary is the same for me, with or without `--with-expat`: 
```
configure: Configured to build cadaver 0.24:

  Install prefix:        /opt/local
  Compiler:              /usr/bin/clang
  Neon library:	         library in /opt/local (0.32.5)
  XML Parser:            using whatever neon uses
  SSL library:		 SSL is supported by neon
  Internationalization:  Built using native support
  GNU readline support:	 enabled
```